src/bloqade/qasm2/_wrappers.py

```diff
@@ -59,7 +59,7 @@ def reset(qarg: Qubit) -> None:
 
 
 @wraps(core.Measure)
-def measure(qarg: Qubit, cbit: Bit) -> None:
+def measure(qarg: Qubit | QReg, cbit: Bit | CReg) -> None:
```

Comment:
so you can fix the type hinting by adding the following separately

```python
from typing import overload

@overload
def measure(qarg: Qubit, cbit: Bit) -> None: ...

@overload
def measure(qarg: QReg, cbit: CReg) -> None: ...
```

Reply:
this is what it looks like in IDE

<img width="1213" alt="image" src="https://github.com/user-attachments/assets/4e175e0b-5142-4fb9-9671-75e6be0d2b45" />

Reply:
@Roger-luo I had already done that following your message on slack. I just forgot to push the commit yesterday :sweat_smile: